subprojects/build-cache/src/main/java/org/gradle/caching/internal/controller/service/BuildCacheServiceRole.java

```diff
@@ -26,7 +27,7 @@
     private final String displayName;
 
     BuildCacheServiceRole() {
-        this.displayName = name().toLowerCase();
+        this.displayName = TextUtil.toLowerCaseUserLocale(name());
```

Comment:
It doesn't make a difference here, but this is probably intended to be English locale only.